packages/components/snackbar/src/snackbar.ts

```diff
@@ -49,6 +50,11 @@ export const snackbarProps = buildProps({
     required: false,
     default: 0,
   },
+  hasCloseButton: {
+    type: Boolean,
+    required: false,
+    default: false,
```

Comment:
correct me if I'm wrong but right now, there is always a close button, right ?
So to avoid a breaking change, the default here should be true.

Reply:
Maaaaaaaaah that's true...